meta-ov/recipes-apps/ovmenu-ng-skripts/files/upload-all.sh

```diff
@@ -2,13 +2,10 @@
 
 USB_PATH="/usb/usbstick/openvario/upload"
 UPLOAD_PATH="/home/root/.xcsoar"
-if [ -z "$(ls $USB_PATH/* 2>/dev/null)" ]; then
+if [ -z "$(ls ${USB_PATH}/* 2>/dev/null)" ]; then
         echo "No files found !!!"
 else
-        for uploadfile in $(find $USB_PATH -type f); do
-                echo $uploadfile
-                cp $uploadfile $UPLOAD_PATH
-        done
+        cp -rv "${USB_PATH}/." "${UPLOAD_PATH}"
```

Comment:
Why the `-v`? It wasn't there before, is it part of fixing something?

Reply:
previously each file was echoed to the gui box that ovmenu-ng invokes, to display what was copied to the user. This emulates that echo with a single command.

Reply:
ovmenu.ng expects the output of the program to be verbose. since we are no longer echoing, -v will add output which is displayed to the user.